app/frontend/src/components/bcgov/BCGovNavBar.vue

```diff
@@ -1,20 +1,25 @@
 <template>
   <nav v-if="!hideNavBar" class="navigation-main">
-    <div class="nav-holder">
-      <ul>
-        <li>
-          <router-link :to="{ name: 'About' }">About</router-link>
-        </li>
-        <li>
-          <router-link :to="{ name: 'UserForms' }">My Forms</router-link>
-        </li>
-        <li>
-          <router-link :to="{ name: 'FormCreate' }">Create a New Form</router-link>
-        </li>
-        <li v-if="isAdmin">
-          <router-link :to="{ name: 'User' }">User (TBD)</router-link>
-        </li>
-      </ul>
+    <div class="container py-0">
```

Comment:
this is the only container i've added, although the diff makes it look like i changed more